layouts/fill-class-detail.vue

```diff
@@ -23,30 +24,83 @@ const [classType] = defineField("classType");
 const [format] = defineField("format");
 const [capacity] = defineField("capacity");
 const [dates] = defineField("dates");
+const [imageUrl] = defineField("imageUrl");
 
 const onSubmit = handleSubmit((values: any) => {
     // TODO: send data to backend
     console.table("values: ", values);
 });
```

Comment:
Should it add logic send data to backend?

Reply:
ยังไม่ทำ

Reply:
ทำมัยมั่ยทำ

Reply:
pr นี้เอามาเสียบก่อนให้รู้ว่าใช้ได้

Reply:
okie